libraries/abstractions/common_io/include/iot_spi.h

```diff
@@ -89,10 +88,10 @@ typedef enum
  */
 typedef enum
 {
-    eSPISetMasterConfig,  /** Sets the configuration of the SPI master. */
-    eSPIGetMasterConfig,  /** Gets the configuration of the SPI master. */
-    eSPIGetTxNoOfbytes,   /** Get the number of bytes sent in write operation. */
-    eSPIGetRxNoOfbytes,   /** Get the number of bytes received in read operation. */
+    eSPISetMasterConfig, /** Sets the configuration of the SPI master. */
```

Comment:
Also, SetMasterConfig and GetMasterConfig comment should indicate datatype being passed.

Reply:
ok, I document it in the ioctl function. I will also add it here.